packages/web-components/src/components/ic-dialog/ic-dialog.css

```
@@ -0,0 +1,131 @@
+@import "../../global/normalize.css";
+
+/** 
+  * @prop --ic-z-index-dialog: z-index of dialog 
+  */
+
+:host {
+  position: fixed;
+  top: 0;
+  left: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  min-height: 100% !important;
+  background: rgb(0 0 0 / 60%);
+  z-index: var(--ic-z-index-dialog);
+  opacity: 0;
+  transition: opacity var(--ic-easing-transition-slow);
+}
+
+:host(.fade-in) {
+  opacity: 1;
+  transition: opacity var(--ic-easing-transition-slow);
+}
+
+:host(.hidden) {
+  display: none;
+}
+
+.dialog {
+  background-color: white;
+  border: 1px solid var(--ic-architectural-300);
+  border-radius: var(--ic-border-radius);
+  padding: var(--ic-space-md);
+  transform: translateY(-3rem);
+  transition: transform 1000s;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+}
+
+:host(.fade-in) .dialog {
+  transform: translateY(0);
+  transition: transform var(--ic-easing-transition-slow);
+}
+
+.small {
+  width: 50%;
+  max-width: 25rem;
+  min-height: 11rem;
+  max-height: 70vh;
+}
+
+.medium {
+  width: 70vw;
+  max-width: 44rem;
+  min-height: 12.5rem;
+  max-height: 70vh;
+}
+
+.large {
+  width: 90vw;
+  max-width: 62.5rem;
+  min-height: 12.5rem;
+  max-height: 90vh;
+}
+
+.heading-area {
+  display: flex;
+  margin-top: calc(-1 * var(--ic-space-xs));
+}
+
+.close-icon {
+  margin-left: auto;
+}
+
+.close-icon-svg {
+  display: flex;
+  align-items: center;
+}
+
+.content-area {
+  overflow-y: scroll;
+  -ms-overflow-style: none;
+  scrollbar-width: none;
```

```suggestion
  scrollbar-width: none;
  padding: var(--ic-space-xs);
  margin: calc(-1 * var(--ic-space-xs));
```
I think if you add the above padding & margins, it should allow for focus indicator to appear for slotted elements (eg textfield, checkbox, radio etc) without being cropped. You would then not need to set margins on the checkbox group in the slotted story